src/main/graphql-server/middleware.js

```diff
@@ -41,7 +42,15 @@ const server = new ApolloServer({
   },
 })
 
+const btoa = arg => {
+  return Buffer.from(arg).toString('base64')
+}
 app.use(renderer)
+app.use('/graphql', (req, res, next) => {
+  const auth = btoa('admin:admin')
```

Comment:
We shouldn't be hardcoding these creds in the middleware file, as this file can get bundled with prod and will inevitably break auth. I believe @gjvera is working on something related to this.

Reply:
I think we need to find a way to set the auth for the playground without touching the backend.